hack/test

```diff
@@ -23,20 +22,6 @@ if [ -n "$CACHE_TO" ]; then
   done
 fi
 
-if [ "$#" == 0 ]; then TEST_INTEGRATION=1; fi
-
-while test $# -gt 0; do
-  case "$1" in
-    integration)
-      TEST_INTEGRATION=1
-      ;;
-    *)
-      echo "unknown arg $1"
-      ;;
-  esac
-  shift
-done
```

Comment:
I think I'd like to keep this if possible, we still default to `integration` if nothing else is specified.

I think we're going to need to create another suite of integration tests at some point to test the complexities of driver creation, e.g. there are a lot of options for setting up the kubernetes driver. These tests don't test the `commands` package directly, so it would make more sense to split it out, so we can run them in parallel etc.